src/platform/ESP32/OTAImageProcessorImpl.cpp

```diff
@@ -280,6 +280,37 @@ esp_err_t OTAImageProcessorImpl::DeltaOTAWriteCallback(const uint8_t * buf, size
 }
 #endif // CONFIG_ENABLE_DELTA_OTA
 
+#if defined(CONFIG_AUTO_UPDATE_RCP) && defined(CONFIG_OPENTHREAD_BORDER_ROUTER)
+esp_err_t OTAImageProcessorImpl::ProcessRcpImage(intptr_t context, const uint8_t * buffer, uint32_t bufLen)
```

Comment:
Why do we use a context for an OTAImageProcessorImpl in a non-static function? Please use `this` instead.